src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java

```diff
@@ -253,4 +254,19 @@ public void deassignCoach(long id) {
             throw new TestNotFoundException();
         }
     }
+
+    @Override
+    public void checkRights(Test test) {
```

Comment:
checkRights name is too wide while it checks only test owner. Let's rename to smth more concreate